### ISSUE TITLE
Document accessibility issues with stacked bar chart with text overlays

### ISSUE THREAD

**Issue by benjiallen** [benjiallen/accessible-python-dev-survey-2020#4]
Review the stacked bar chart with text overlays (see Figure 1) used within the [python developers survey](https://www.jetbrains.com/lp/python-developers-survey-2020/). See the section entitled "Python 3 vs Python 2".

Figure 1
<img width="385" alt="2021-04-09_20-49-09" src="https://user-images.githubusercontent.com/16546334/114281840-adbd0e00-9a0e-11eb-9ef6-44cc231a6dc4.png" alt="stacked bar chart with text overlays used in survey results">

The review should be completed with respect to the WCAG 2.1 level A and AA guidelines.

In addition to the accessibility review, consider these questions:

* Could this visualization be easily represented in a data table?
* Does using a data table make the experience more or less efficient? Would important visual data, like trends, be lost in a data table presentation?
* Would an alternative visualization do a better job of conveying the data? If so, why?
* Does the chart title and supporting text do a good enough job in explaining the data within the visualization?
